flipperdevices/flipperzero-firmware
# Support reseting iCx cards

# What's new

- Support resetting iClass cards to iCL/iCS/iCE (https://github.com/iCopy-X-Community/icopyx-teardown/blob/master/tags/README.md#icl)

# Verification 

- Scan iClass card with standard key, see new options.
- Choose one of the options
- Use card with iCopyX (successfully)

# Checklist (For Reviewer)
